### DISCUSSION TITLE
[Roadmap] Q4 2021

### DISCUSSION THREAD

**Discussion by emplums** [primer/react#961] [General]
## Goals
1. **Support**  Support our high priority teams building with Primer React Components at GitHub
2. **Coverage** Increase coverage of Primer React Components
3.  **DX** Build tools that improve the developer experience of working with Primer Components

### Automate props documentation

**Issue:** 
#### Impact
Automating props documentation would increase the reliability of our documentation and improve maintainer experience by making sure that props docs are always up to day.

Related Goals: Support + DX
**Status**: Research

### Intellisense for system props

**Issue:** 
#### Impact
Adding Intellisense support for system props would greatly improve developer experience by speeding up the time it takes to find the correct value for props and scan our theme within the comfort of your own text editor.

Related Goals:  DX
**Status**: Research
### Datepicker

**Issue:** 
#### Impact
Supports teams building with PRC that are introducing Datepickers into their applications.

Related Goals:  Support + Coverage
**Status**: Research
### User Research + Streamlined Feedback Process

**Issue:** 
#### Impact
Now that Primer React Components is heavily used in other projects, let's take some time to talk to engineers at GitHub using Primer React Components to gather feedback on what is and is not working. Some questions I have in mind that I would like to gather more feedback on:
- Ergonomics of system props
- Needed components
- Roadblocks to contributing to PRC

Related Goals:  Support + DX + Coverage
**Status**: Research
#### Scope

cc @colebemis @BinaryMuse @broccolini

**Comment by aaftab-vijapura** [comment]
intresting👍
